cmd/bnscli/cmd_distribution_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/iov-one/weave/cmd/bnsd/app"
+	"github.com/iov-one/weave/weavetest/assert"
+	"github.com/iov-one/weave/x/distribution"
+)
+
+func TestCmdResetRevenue(t *testing.T) {
+	recipientsPath := mustCreateFile(t, strings.NewReader(`seq:foo/bar/1,3
```

Comment:
Very nice use of the Address shorthands here, not just hex.

Reply:
First I've spent a few minutes trying to provide a valid hex address. Then I have realized that we support many formats :tada: